scripts/ci/configure.sh

```diff
@@ -30,3 +30,5 @@ fi
 
 # Configure Audacity
 cmake "${cmake_args[@]}"
+
+conan remove "*" -s -b -fs
```

Comment:
Has the `-s` option been given twice? BTW, I tend to prefer long options where available, except maybe with the most common commands that everybody knows like `ln -s`, etc.

Reply:
I will replace with the full options